README.md

```diff
@@ -100,6 +100,11 @@ if no file is found). It is mutually exclusive with `:load-fn!` and
 will be ignored in case both are present
 
 * `:src-paths`  a vector of paths containing source files
+
+* `:cache` - a map containing two optional values: the first, `:path`, indicates
+the path of the cached files. The second, `:search-src-paths?`, indicates
```

Comment:
With ? of course if it is a Boolean, a question is, does it work if I want both? Look in a custom path AND in src-paths?

On January 19, 2016 12:12:25 PM PST, Tomek <notifications@github.com> wrote:
>> @@ -100,6 +100,11 @@ if no file is found). It is mutually exclusive
>with `:load-fn!` and
>>  will be ignored in case both are present
>>  
>>  * `:src-paths`  a vector of paths containing source files
>> +
>> +* `:cache` - a map containing two optional values: the first,
>`:path`, indicates
>> +the path of the cached files. The second, `:search-src-paths?`,
>indicates
>
>ok for `:src-path-lookup`
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ScalaConsultants/replumb/pull/127/files#r50168068

-ar